src/components/research-form-wizard/FormPages/FollowUp.tsx

```diff
@@ -0,0 +1,197 @@
+  gap: 10px;
+  font-size: 12px;
+  color: #555;
+  margin: 10px 0;
+`;
+
+const TermsCheckbox = styled.input`
+  appearance: none;
+  border: 1px solid lightgray;
+  border-radius: 4px;
+  width: 16px;
+  height: 16px;
+  position: relative;
+  cursor: pointer;
+  flex-shrink: 0; /* Prevent the checkbox from shrinking */
+
+  &:checked {
+    border-color: #6e56cf;
+    background-color: #aa99ec;
+  }
+
+  &:checked::before {
+    content: '✓';
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: white;
+    font-size: 10px;
+  }
+`;
+
+const FollowUpComponent = () => {
+  const set = useResearchWizardStore((state) => state.set);
+  const [interest, setInterest] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [agreed, setAgreed] = useState<boolean>(false);
+
+  useEffect(() => {
+    const validEmail = isValidEmail(email);
+    let isDisabled = true;
+    if (validEmail && agreed) {
+      isDisabled = false;
+    }
+
+    set({
+      nextDisabled: isDisabled,
+      currentStepSubmission: { optIn: interest, emailAddress: email, agreedToTerms: agreed },
+    });
+  }, [interest, email, agreed, set]);
+
+  return (
+    <>
+      <Header>
+        <Title>Help us with Research</Title>
+      </Header>
+      <Description>
+        If you submit your email, you consent to be contacted and may be asked to participate in a future paid study
+        regarding the Near Ecosystem. Your feedback will be sincerely appreciated as it ensures we are building the
+        right things to serve your needs.
+      </Description>
+      <SubTitle>Are you interested?</SubTitle>
+      <RadioGroup>
+        <RadioLabel>
+          <RadioInput
+            type="radio"
+            name="interest"
+            value="Yes, please!"
+            checked={interest === 'Yes, please!'}
+            onChange={() => setInterest('Yes, please!')}
+          />
+          Yes, please!
+        </RadioLabel>
+        <RadioLabel>
+          <RadioInput
+            type="radio"
+            name="interest"
+            value="No thanks"
+            checked={interest === 'No thanks'}
+            onChange={() => setInterest('No thanks')}
+          />
+          No thanks
+        </RadioLabel>
+      </RadioGroup>
+      <EmailInput
+        type="email"
+        placeholder="name@email.com"
+        visible={interest === 'Yes, please!'}
+        value={email}
+        onChange={(e) => setEmail(e.target.value)}
+      />
+      <Description>Your email will be only used for research outreach and will never be shared.</Description>
+      <TermsLabel visible={interest === 'Yes, please!'}>
+        <TermsCheckbox type="checkbox" checked={agreed} onChange={() => setAgreed(!agreed)} />
+        By checking this box you hereby agree that you have read and agree to our Terms of Use and Privacy Policy
```

Comment:
Maybe we should add links to terms and privacy pages here?

Reply:
@jackson-harris-iii  https://dev.near.org/terms & https://dev.near.org/privacy